src/test/scala/org/kidsfirstdrc/dwh/testutils/external/GnomadV311Input.scala

```diff
@@ -0,0 +1,18 @@
+package org.kidsfirstdrc.dwh.testutils.external
+
+case class GnomadV311Input(
+                            contigName: String = "chr1",
+                            start: Long = 999,
+                            end: Long = 1009,
+                            referenceAllele: String = "A",
+                            alternateAlleles: List[String] = List("C"),
+                            qual: Double = 0.5,
+                            names: List[String] = List("BRAF"),
+                            INFO_AC: List[Long] = List(1),
```

Comment:
Since we are reading the vcf with the option
`.option("flattenInfoFields", "true")`
these fields should not be arrays anymore